test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/disposaldetails/DisposalDetailsControllerSpec.scala

```diff
@@ -102,6 +109,60 @@ class DisposalDetailsControllerSpec
   ): (SessionData, FillingOutReturn, DraftSingleDisposalReturn) =
     sessionWithDisposalDetailsAnswers(Some(answers), disposalMethod)
 
+  def fillingOutReturnForIndividual(
+    disposalMethod: DisposalMethod,
+    subscribedDetails: SubscribedDetails
+  ): (FillingOutReturn, DraftSingleDisposalReturn) = {
+    val answers = sample[CompleteSingleDisposalTriageAnswers].copy(
+      disposalMethod = disposalMethod
+    )
+
+    val draftReturn = sample[DraftSingleDisposalReturn].copy(
+      triageAnswers = answers
+    )
+    sample[FillingOutReturn].copy(draftReturn = draftReturn, subscribedDetails = subscribedDetails) -> draftReturn
+  }
+
+  def sessionWithDisposalDetailsAnswers(
+    answers: Option[DisposalDetailsAnswers],
+    disposalMethod: DisposalMethod,
+    subscribedDetails: SubscribedDetails
+  ): (SessionData, FillingOutReturn, DraftSingleDisposalReturn) = {
+    val (journey, draftReturn) = fillingOutReturnForIndividual(disposalMethod, subscribedDetails)
+    val updatedDraftReturn     = draftReturn.copy(disposalDetailsAnswers = answers)
+    val updatedJourney         = journey.copy(draftReturn = updatedDraftReturn)
+    (
+      SessionData.empty.copy(journeyStatus = Some(updatedJourney)),
+      updatedJourney,
+      updatedDraftReturn
+    )
+
+  }
+
+  def sessionWithDisposalDetailsAnswersForIndividual(
+    answers: DisposalDetailsAnswers,
+    disposalMethod: DisposalMethod
+  ): (SessionData, FillingOutReturn, DraftSingleDisposalReturn) = {
+
+    val subscribedDetails = sample[SubscribedDetails].copy(
+      name = Right(sample[IndividualName])
+    )
+
+    sessionWithDisposalDetailsAnswers(Some(answers), disposalMethod, subscribedDetails)
```

Comment:
I can't see how you are ensuring the userType in the session is UserType.Individual so you may end up with a sampled UserType such as UserType.Organisation or UserType.Agent in your session - we need to do more than just set the name to match our userType, this may mean passing a new param in to `sessionWithDisposalDetailsAnswers` such as `userType: UserType` and set it in this call to `UserType.Individual` then you can add in the necessary `userType = Some(userType)` to the session when copying extra vals in - best to look at other specs that handle this where we use methods like `setAgentReferenceNumber(userType)` because we want to control this also